internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -2217,12 +2214,10 @@ Segment getSegment(int partition, boolean sendEndOfSegment) {
           long currentSegmentStartTime = segmentsStartTimeArray[partition];
           if (currentSegmentStartTime != -1
               && LatencyUtils.getElapsedTimeFromMsToMs(currentSegmentStartTime) > maxElapsedTimeForSegmentInMs) {
-            String msg = "Segment " + currentSegment + " has been open for more than" + maxElapsedTimeForSegmentInMs
-                + " ms, ending it and starting a new one.";
-
-            if (!REDUNDANT_LOGGING_FILTER.isRedundantException(msg)) {
-              logger.info(msg);
-            }
+            logger.info(
```

Comment:
Not sure if this log would be helpful; for store that has thousands of partitions, this might end up printing thousands of logs in a short time.

Reply:
Removed the log line. Can you help take a look? Thank you!